javascripts/quiz.js

```diff
@@ -0,0 +1,26 @@
+console.log ("two robots enter, one robot leaves");
+
```

Comment:
This is your only js page. there isn't any logic in your pull request. Make sure you wrote all your code in you quiz branch